chalice/deploy/swagger.py

```diff
@@ -128,6 +129,11 @@ def _generate_route_method(self, view):
             'x-amazon-apigateway-integration': self._generate_apig_integ(
                 view),
         }  # type: Dict[str, Any]
+        if view.view_function.__doc__:
+            doc_lines = inspect.getdoc(view.view_function).splitlines()
```

Comment:
How come we use `__doc__` in the if statement and `inspect.getdoc(...)`to get the actual docs? No need to import inspect if we just use `__doc__` in both places. It looks like you manually clean up the string anyway so you aren't really getting any benefit from the fact that `getdoc` calls `cleandoc` as far as I can tell.

Reply:
I use ```__doc__``` because it's a quick check if a docstring exists at all. If it does exist, I need to get the docstring with the indention at the function level. I saw that the ```getdoc()``` version returns corrected indentation. I'm open to other ways to get it, but I found that ```getdoc()``` seemed to be the best approach.

Reply:
How about just:

```python
doc_lines = inspect.getdoc(....)
if doc_lines:
    ...
```
Reads cleaner to me and I'm not worried about the small performance difference during a deployment.

Reply:
sounds good

Reply:
Just want to confirm again that this looks better to you:

```python
        docstring = inspect.getdoc(view.view_function)
        if docstring:
            doc_lines = docstring.splitlines()
            current['summary'] = doc_lines[0]
            if len(doc_lines) > 1:
                current['description'] = '\n'.join(doc_lines[1:]).strip('\n')
```